### PR TITLE
add recipe for ninrod/evil-string-inflection package

### PR DESCRIPTION
**Pull request by ninrod**
### Brief summary of what the package does

provides an evil operator to cycle text objects through various case formats: kebab, camel, snake, UPPER.

### Direct link to the package repository

https://github.com/ninrod/evil-string-inflection

### Your association with the package

I'm the author

### Relevant communications with the upstream package maintainer

**none needed**

### Checklist

Please confirm with `x`:

- [x ] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses). 
- [ x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [ x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x ] My elisp byte-compiles cleanly
- [ x] `M-x checkdoc` is happy with my docstrings
- [ x] I've built and installed the package using the instructions in [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: recipes/snakecamelfy [hunk 1]
```diff
@@ -0,0 +1 @@
+(exato :repo "ninrod/snakecamelfy" :fetcher github)
```
Comment (tarsius) [inline recipes/snakecamelfy]: `exato` has to be replaced.

Comment (ninrod) [reply recipes/snakecamelfy]: :man_facepalming:

Comment (ninrod) [reply recipes/snakecamelfy]: Ok, done!

**Comment by ninrod** [comment]
@tarsius, should be fine now.

**Comment by tarsius** [comment]
The build error is due to an unrelated change, which is fixed now. Please rebase your commit so that the tests can be run for your change. If you use Magit you can do so using <kbd>f u</kbd> to fetch, <kbd>r u</kbd> to rebase, and <kbd>P - f p</kbd> to force push.

By the way you should also do that fixing an issue in your recipe submission. So instead of adding a second commit "fix copy/paste leftover", you should just <kbd>c e</kbd> to amend and <kbd>P - f p</kbd> to force push. @purcell that's what you would prefer, right?

**Comment by ninrod** [comment]
@tarsius and @purcell, I've implemented the rebase and squashed the commits. but it seems that there's an issue in travis regarding the build [path](https://travis-ci.org/melpa/melpa/builds/308348271#L1036).

**Comment by tarsius** [comment]
Turns out my fix was not complete... still working on it.

**Comment by tarsius** [comment]
Please try again it should work now.

**Comment by ninrod** [comment]
@tarsius, do you need anything else from me regarding this pull?

**Comment by tarsius** [comment]
No, the tests pass, which means the things that I broke are no longer broken and also that nothing that could be detected by the tests is broken in your submission :wink:

**Comment by purcell** [comment]
Great! Same comments here as in #5149, re. package naming and key customisation.

**Comment by purcell** [comment]
Also, you might consider using the [string-inflection package](`https://github.com/akicho8/string-inflection/blob/e644c8bf2a5e36ddadcfe4de657588c00fd368dd/string-inflection.el) to avoid reinventing this code.

**Comment by ninrod** [comment]
> Also, you might consider using the string-inflection package to avoid reinventing this code.

This was a game changer. I had to rewrite and rename the whole package, got more functionality than I ever hoped and in the process end up with just 4 lines of code. Talk about reuse. 👍 

thank you very much @purcell. I'd appreciate a new review.

**Pull request merged**